### ISSUE TITLE
Deletion based on file type from all the sub directories

### ISSUE THREAD

**Issue by aquatiko** [source-aasf/AutoDel#2]
can be implemented, but needs an efficient directory traversing method

**Comment by ab-anand** [comment]
Yeah, we thought of that. This project was created just out of boredom so we never implemented it :p  

Also, in that case:-
* For mp3 files, u can delete on the basis of whether it's a 320kbps file or 128kbps
* For videos files, u can use pixels

So yeah, there can be many more such implementations and you surely wld need an effective strategy. Though you just need to make it faster than other such software if any XD
